.github/workflows/android-release.yml

```diff
@@ -0,0 +1,83 @@
+#This is a workflow that creates, downloads, uploads and releases APK-file.
+name: APK Workflow
+
+#Event triggers
+#execute the jobs everytime when a pull_request to the main-branch is created
+on:
+  push:
+    branches: main
+
+jobs:
+  runtests:
+
+    name: Run Setup Tests
+    runs-on: ubuntu-latest
+    steps:
+
+      #Checkout a Git repository and set up Java JDK
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1.4.3
+        with:
+          java-version: 1.8
+
+      #command: run unit tests
+      - name: Run Build Test
+        #Working directory is the folder with the project
+        working-directory: ./BibBuddy
+        run: bash ./gradlew test --stacktrace
+
+      #command:  Run Java Checkstyle
+      - name: Run Java Checkstyle
```

Comment:
We already run checkstyle in the android.yml workflow when we create a PR to dev. Reason see above.

Reply:
Yes, but as I made a test from my currently created branch to main-branch, there were some checkstyle errors.  see #92 

Before our CI workflow integration, we merged code to dev, which is not already checked.

Reply:
Here is the workflow run:
https://github.com/UniRegensburg/unsere-app-fur-die-universitat-regensburg-bib-buddy/actions/runs/607611443